Cargo.toml

```diff
@@ -81,3 +81,10 @@ test_dir = "0.2.0"
 
 [build-dependencies]
 vergen = { version = "7.4.2", default-features = false, features = ["build", "cargo", "git", "rustc"] }
+
+[profile.release]
+opt-level = 3
+debug = false
+panic = "unwind"
```

Comment:
Aren't `opt-level = 3`, `debug = false` and `panic = "unwind"`, defaults for the release profile? (1). As such, I think it is not necessary to re-specify these; what do you think?

(1) https://doc.rust-lang.org/cargo/reference/profiles.html#release
(2) https://doc.rust-lang.org/cargo/reference/profiles.html#profiles

Reply:
Makes sense! I updated the PR 🐻